### PR TITLE
[zh] Fix broken links, anchors and translate for customize install doc

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Fix broken links, anchors and translate for customize install doc

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/additional-setup/customize-installation/index.md [hunk 1]
```diff
@@ -79,30 +79,30 @@ spec:
...
-1. [Toleration](https://kubernetes.io/zh-cn/docs/concepts/configuration/taint-and-toleration/)
-1. [Strategy](https://kubernetes.io/zh-cn/docs/concepts/workloads/controllers/deployment/)
-1. [Env](https://kubernetes.io/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container/)
-1. [Pod security context](https://kubernetes.io/zh-cn/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
-1. [Volumes and volume mounts](https://kubernetes.io/zh-cn/docs/concepts/storage/volumes/)
+1. [优先级类名称](https://kubernetes.io/zh-cn/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass)
+1. [节点选择器](https://kubernetes.io/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)
+1. [亲和性与反亲和性](https://kubernetes.io/zh-cn/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
+1. [服务（Service）](https://kubernetes.io/zh-cn/docs/concepts/services-networking/service/)
+1. [容忍](https://kubernetes.io/zh/docs/concepts/scheduling-eviction/taint-and-toleration)
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/customize-installation/index.md:101]:
```suggestion
1. [容忍度](https://kubernetes.io/zh/docs/concepts/scheduling-eviction/taint-and-toleration)
```
点过去标题是 容忍度

File: content/zh/docs/setup/additional-setup/customize-installation/index.md [hunk 2]
```diff
@@ -224,15 +224,15 @@ spec:
 
 ## 高级安装定制 {#advanced-install-customization}
 
-### 定制外部 chart 和配置项 {#customizing-external-charts-and-profiles}
+### 定制外部 Chart 和配置项 {#customizing-external-charts-and-profiles}
 
-`istioctl` 的 `install`、 `manifest generate` 和 `profile` 命令可以使用以下任意源来生成 chart 和 配置档：
+`istioctl` 的 `install`、 `manifest generate` 和 `profile` 命令可以使用以下任意源来生成 Chart 和 配置档：
```
Comment (windsonsea) [inline content/zh/docs/setup/additional-setup/customize-installation/index.md:229]:
```suggestion
`istioctl` 的 `install`、 `manifest generate` 和 `profile` 命令可以使用以下任意源来生成 Chart 和配置项：
```
小标题叫  配置项，正文叫 配置档，应该需要统一

**Review by windsonsea — COMMENTED** [review]
good job.

**Review by windsonsea — APPROVED** [review]
Thanks. Those links in the en doc shall be changed too.

That would be a separate job to open a new PR to update the en upstream version.

**Comment by wilsonwu** [comment]
> Thanks. Those links in the en doc shall be changed too.
> 
> That would be a separate job to open a new PR to update the en upstream version.

The links in English doc all work fine, I checked, seems k8s doc site English version has mechanism for compatibility.

**Review by windsonsea — APPROVED** [review]

**Comment by windsonsea** [comment]
Yes, they added many redirects.

**Pull request merged**